src/main/java/techreborn/client/render/entitys/CableCoverEntityRenderer.java

```diff
@@ -0,0 +1,39 @@
+package techreborn.client.render.entitys;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.render.RenderLayer;
+import net.minecraft.client.render.RenderLayers;
+import net.minecraft.client.render.VertexConsumer;
+import net.minecraft.client.render.VertexConsumerProvider;
+import net.minecraft.client.render.block.BlockRenderManager;
+import net.minecraft.client.render.block.entity.BlockEntityRenderDispatcher;
+import net.minecraft.client.render.block.entity.BlockEntityRenderer;
+import net.minecraft.client.util.math.MatrixStack;
+import techreborn.blockentity.cable.CableBlockEntity;
+import techreborn.blocks.cable.CableBlock;
+
+import java.util.Random;
+
+public class CableCoverEntityRenderer extends BlockEntityRenderer<CableBlockEntity> {
```

Comment:
Wonder if this can be done via a FabricBakedModel? Would be more complex however

Reply:
I've tried. There is lack of documentation and examples, so I give up

Reply:
Lack of documentation? The javadocs cover everything and then some?

Reply:
I constantly use decompiled versions and wasn't aware of java docs :(